tasks/credentials.yml

```diff
@@ -5,8 +5,8 @@
 - name: Fetch credentials tokens locally
   synchronize:
     mode: pull
-    src: "{{ openio_keystone_credentials_tokens_key_repository }}"
-    dest: "{{ openio_keystone_tmp_keys_dir }}/credential-keys"
+    src: "{{ openio_keystone_credentials_tokens_key_repository }}/"
```

Comment:
Is that fixing something ?

My tastes are the other way around, to avoid multiple-slashes in case of:
{{ dir_base }}/{{ subdir_1 }}/{{ filename }}
but maybe that's just me...

Reply:
src and dest are directories, that's why we force the ending slash.
In some case without the ending slash, it copies the directory aswell as files and the result is not the one we expect.